.github/actions/boost-install-dep/action.yml

```diff
@@ -0,0 +1,60 @@
+name: 'Install Boost Dependency'
+description: 'Install Boost Dependency using cache when possible'
```

Comment:
did you try installing boost in the sb ? Does it work ?

Reply:
there's already a boost project in the common-sb

Reply:
yes, but did you try it ? Did you try a usd project as well ? Just checking how hard it will be to deliver this in the 2.2 binary release.

Reply:
I guess we'll see? If that's too complicated, we'll drop it.